Sources/Fuzzilli/Evaluation/ProgramEvaluator.swift

```diff
@@ -39,10 +39,12 @@ public protocol ProgramEvaluator: Component {
     /// Import a previously exported state.
     func importState(_ state: Data) throws
 
-    /// Takes previously acquired aspects and a new execution, and returns the aspects of the
-    /// intersection of each.
-    /// Resets any edges found in the execution but not the provided aspects
-    func evaluateAndIntersect(_ program: Program, with aspects: ProgramAspects) -> (ProgramAspects, Bool)?
+    // Resets the provided aspects and executes the program a second time.
+    // If aspects are successfully collected from the second execution, returns
+    // the intersection of the initally provided aspects with the aspects from the 
+    // second execution, and whether or not the aspects from the second execution match
```

Comment:
The `, and whether or not the aspects from the second execution match those from the first` part is no longer the case, is it?